Covinator/Covinator/Pages/PfizerVaccination.cshtml

```diff
@@ -1,8 +1,8 @@
 ﻿@page
 @model Covinator.Pages.PfizerVaccination
 @{
-    ViewData["Title"] = "Home page";
-    var pfizerVaccineDistributionAllocationss = (QuickType.PfizerVaccineDistributionAllocations[])ViewData["PfizerVaccineDistributionAllocations"];
+    ViewData["Title"] = "Pfizer Vaccination";
+    var pfizerAllocations = (QuickType.PfizerVaccineDistributionAllocations[])ViewData["PfizerVaccineDistributionAllocations"];
```

Comment:
Reduced length of variable name.

Reply:
Source:-
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions

Reply:
Variable names should be long enough to be descriptive, but yes, in this case... the words removed are implicit from the context of this page, so safe to remove them.